recipe/build-libadios.sh

```diff
@@ -66,37 +47,38 @@ if [[ "${CONDA_BUILD_CROSS_COMPILATION}" == "1" ]]; then
     export CMAKE_ARGS="${CMAKE_ARGS} -DADIOS2_HAVE_ZFP_CUDA_EXITCODE=0"
 fi
 
-
-cmake              \
-    -S ${SRC_DIR}  \
-    -B build       \
-    ${CMAKE_ARGS}  \
+# shellcheck disable=SC2086
+cmake               \
+    $CMAKE_ARGS     \
+    -S "$SRC_DIR"   \
```

Comment:
```suggestion
    -S "${SRC_DIR}"   \
```
ok if the path would ever include a space (not really on Unix/conda)

Reply:
I was making happy shellcheck, I agree that it is unlikely, however, in unix is possible to have files/dirs with spaces.